http/src/test/resources/reference.conf

```diff
@@ -71,6 +71,13 @@ pubsub {
   non-leo-message-subscriber {
     subscription-name = "nonLeoMessageSubscription"
   }
+
+  kubernetes-monitor {
```

Comment:
is this used?

Reply:
Yeah I added a test to verify `startApp` timeout, so I wanted a test config with a shorter timeout

Reply:
oh I see....I thought this is new config you added, and didn't see it being used anywhere in this PR